src/entrypoint/nsinstance.go

```diff
@@ -427,47 +393,6 @@ func env(preserve []string, override ...string) []string {
 	return r
 }
 
-// setuppty creates a PTY suitable for use with interceptsl.
-func setuppty() (*PTY, error) {
```

Comment:
Just noticed I forgot to use this function during instance creation (which would explain a bit of the odd behavior I noticed with the PTY). Could you put it back and change `OpenPTY` to `setuppty` in `Run`?